ldap/servers/plugins/replication/repl5_agmt.c

```diff
@@ -95,6 +98,14 @@ typedef struct repl5agmt
     const Slapi_DN *dn;                    /* DN of replication agreement entry */
     const Slapi_RDN *rdn;                  /* RDN of replication agreement entry */
     char *long_name;                       /* Long name (rdn + host, port) of entry, for logging */
+#define SID_SUPPORT_UNKNOWN 0
+#define SID_SUPPORT_NO      1
+#define SID_SUPPORT_YES     2
```

Comment:
Just curious. Why bothering checking if control is supported ?
You could systematically send the control with CRITICAL flag unset and the consumer server will ignore it if it is not supported